warehouse/admin/templates/admin/users/detail.html

```diff
@@ -356,18 +512,16 @@ <h3 class="card-title">Emails</h3>
                 </div>
                 {% endfor %}
               </div>
-            </div>
 
             <div class="card-footer">
               <div class="form-group">
                 <div class="col-sm-offset-10 col-sm-2">
-                  <button type="submit" class="btn btn-danger" title="{{ 'Submitting requires superuser privileges' if not perms_admin_users_write }}" {{ "disabled" if not perms_admin_users_write }}>Submit</button>
+                  <button type="submit" action="email_form" class="btn btn-danger" title="{{ 'Submitting requires superuser privileges' if not perms_admin_users_email_write }}" {{ "disabled" if not perms_admin_users_email_write }}>Submit</button>
                 </div>
               </div>
             </div>
           </form>
         </div>
-      </div> <!-- .card -->
```

Comment:
One of these div changes made the Name field appear disjoint from the rest of the form

<img width="330" alt="Screenshot 2024-07-15 at 12 11 04" src="https://github.com/user-attachments/assets/93dad278-90c0-44d8-8193-87205d9ee9ac">

Not super important, but wanted to note the visual oddity.

Reply:
That's not a regression FWIW, it was like that before!

Reply:
The change makes the "Submit" button part of the Card footer, which isolates the "Name", but it's no biggie - it all still works just fine. I'll poke at it some other time.

Reply:
that's also not introduced by these changes, this card has been goofed for some time:

<img width="1066" alt="Screenshot 2024-07-15 at 1 19 43 PM" src="https://github.com/user-attachments/assets/be395b39-cfa2-4ccd-96d7-af9c84a65168">